CefSharp.BrowserSubprocess.Core/CefAppUnmanagedWrapper.h

```diff
@@ -29,6 +29,8 @@ namespace CefSharp
             bool _focusedNodeChangedEnabled;
             bool _legacyBindingEnabled;
             bool _jsBindingApiEnabled = true;
```

Comment:
Storing these on a per browser basis would provide greater flexibility, we could create a class that has a lookup by browser id.